README.md

```diff
@@ -158,14 +155,14 @@ merged into the RFC repository as a markdown file. At that point the RFC is
 [The RFC life-cycle]: #the-rfc-life-cycle
 
 Once an RFC becomes "active" then authors may implement it and submit the
-feature as a pull request to the Rust repo. Being "active" is not a rubber
-stamp, and in particular still does not mean the feature will ultimately be
+change as a pull request to the corresponding Sawtooth repo. Being "active" is not a rubber
```

Comment:
You changed "feature" to "change" in this paragraph, but not in the following content. See lines 166 and 203 for examples.

Reply:
This has been changed, the remaining 'feature' is intentional.